### ISSUE TITLE
Konsole now supports DECCURSOR escape sequences

### ISSUE THREAD

**Issue by sassanh** [neovim/neovim#8327]
Regarding this issue: https://github.com/neovim/neovim/issues/6403 and a few others I've seen in neovim issues:

Konsole now supports setting cursor style with xterm escape sequences in this commit:
7c2a1164be14ae912d166ee79d07cc45cb5e3aba

So the problem with ssh sessions in Konsole should be solved and now Konsole is closer to `xterm-256`.
Just wanted to inform it but couldn't comment on the mentioned issue because it was closed.

**Comment by justinmk** [comment]
But then we still need some way to detect konsole version, or other way of detecting support. Most users will be using an ancient version for years.

**Comment by sassanh** [comment]
Yeah you're right. But now at least there's an option.

When I use Konsole I set `$KONSOLE_PROFILE_NAME` in ssh sessions so that neovim works alright. But now if someone googles `konsole neovim q` and hopefully finds this thread he can either set that variable or upgrade his Konsole like I did yesterday. But I agree that it doesn't change anything for neovim codebase as many use ancient versions of Konsole. I just wanted to comment it in the main issue but comments were closed there so I created another issue.
